src/index.css

```diff
@@ -196,26 +204,6 @@ a {
   line-height: 1.3em;
 }
 
-span.pill {
```

Comment:
Unread message counter is just a bare number now: 
![image](https://user-images.githubusercontent.com/1172179/213419609-4ad79055-1cb9-4c25-8494-bab246380bfa.png)